staging/src/k8s.io/component-base/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Purpose
 
-Implement KEP 32: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/0032-create-a-k8s-io-component-repo.md
+Implement KEP 32: https://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md
 
```

Comment:
@christopherhein  Thanks for your fix
would yu like change the link to `https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md` ?

Reply:
Prefer that over the short link? All ends up the same place

Reply:
@stewart-yu done. 👍